oka/chap4/knock30.py

```diff
@@ -0,0 +1,28 @@
+def maping():
+    with open("neko.txt.mecab", encoding="utf-8") as f:
```

Comment:
```suggestion
    with open("neko.txt.mecab", "r", encoding="utf-8") as f:
```

 ファイルオープンのモードはあったほうがいいと思うな